Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -3360,6 +3391,36 @@ - (void)_endAnimatedResize
         [self _didCompleteAnimatedResize];
 }
 
+- (void)_beginLiveResize
+{
+    WKWEBVIEW_RELEASE_LOG("%p (pageProxyID=%llu) -[WKWebView _beginLiveResize]", self, _page->identifier().toUInt64());
+
+    _liveResizeParameters = { { self.bounds.size.width, self.scrollView.contentOffset } };
```

Comment:
Do we want to allow repeated `_beginLiveResize` without ever calling `_endLiveResize`?  Should we maybe bail early if `_liveResizeParameters` is set?

Reply:
Sure! With a log. Will do